gef-ui/views/templates/name-application.njk

```diff
@@ -0,0 +1,32 @@
+{% extends "index.njk" %}
+{% import '_macros/success-message.njk' as success %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% block pageTitle %}Confirm eligiblity (mandatory criteria){% endblock %}
+
+{% block content %}
+
+  {% if successMessage %}
+    {{ success.message(
```

Comment:
Could we use `render` so it reads

```
successMessage.render( ...
```

It's a good pattern imo, especially with other component names. E.g if we have "table" component it would read `table.render`

Reply:
Going to remove this as its not needed :)